contrib/uwsgi.ini

```diff
@@ -0,0 +1,29 @@
+[uwsgi]
```

Comment:
Going to remove this file?

Reply:
Yes, going to remove the entire `contrib` directory and a smattering of other vestigial files all in another pass soon. Thanks! :)